drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -15,7 +15,7 @@ if ETH_SAM_GMAC
 config ETH_SAM_GMAC_QUEUES
 	int "Number of active hardware TX and RX queues"
 	default 1
-	range 1 $(dt_node_int_prop_int,/soc/ethernet@40050088,num-queues)
+	range 1 $(dt_node_int_prop_int,eth-0,num-queues)
```

Comment:
i assume the address for /soc/ethernet@40050088 differs on some other soc?

Reply:
Yes, SAM 4E, SAM E5X, SAM E/V7X all have different addresses.